### DISCUSSION TITLE
Schema not shown in pgAdmin4

### DISCUSSION THREAD

**Discussion by peterfutaro** [PostgREST/postgrest#3014] [Q&A] [answered]
Hi there,

I was following the tutorial of postgREST in https://postgrest.org/en/stable/tutorials/tut0.html
When I want to check on my learning result via pgAdmin4, I couldn't see the "api" schema created from the tutorial steps.

![275531378-e5682651-5f33-4a56-b47c-3bd6f6b980b6](https://github.com/PostgREST/postgrest/assets/76289958/0ab1447f-9300-4263-bb27-1e07bb9ffd2f)

![275531708-ec633eff-ef88-4afe-995b-2b0b367de8f3](https://github.com/PostgREST/postgrest/assets/76289958/9db1a877-7051-40bf-9b31-8f0cc91e4a3a)

I believe I am using the same "postgres" user in both (when doing tutorial, and in my pgAdmin4)

Can anyone tell me what I have missed ?

Thank you.

**Comment by laurenceisla** [comment]
Hi! Are you using the same Docker database than in the tutorial? Verify that your PgAdmin is connected to your Docker DB instead of the local one, you'll need to register a new connection if that's the case.

**Comment by wolfgangwalther** [comment, marked as answer]
The screenshot says "Databases (2)". Which is the other database in that cluster?
